src/test/librbd/fsx.cc

```diff
@@ -1794,6 +1827,103 @@ do_punch_hole(unsigned offset, unsigned length)
 	memset(good_buf + max_offset, '\0', max_len);
 }
 
+unsigned get_data_size(unsigned size)
+{
+	unsigned i;
+	unsigned hint;
+	unsigned max = sqrt((double)size) + 1;
+	unsigned good = 1;
+	unsigned curr = good;
+
+	srandom(time(NULL));
```

Comment:
```random``` has already been initialized -- no need to keep re-initializing it. In fact, for test repeatability we rely on the fact that we can pass the seed value and recreate a test run exactly.